pkg/cmd/server/api/v1/types.go

```diff
@@ -143,6 +143,14 @@ type MasterConfig struct {
 	// omitted) and controller election can be disabled with -1.
 	ControllerLeaseTTL int `json:"controllerLeaseTTL"`
 
+	// AdmissionPluginOverride is a list of admission control plugins that will be installed
+	// on the Kubernetes master. Order is significant. If empty, a default list of plugins is used.
+	AdmissionPluginOverride []string `json:"admissionPluginOverride,omitempty"`
```

Comment:
We said we would have 2 (which we do). I'm fine with one going in MasterConfig and the other in KubernetesMasterConfig ... but isn't that contradictory to putting them in their own struct?